modules/fb_instant_articles_views/views/fb_instant_articles_views.views.inc

```diff
@@ -20,6 +21,16 @@ function fb_instant_articles_views_views_data_alter(&$data) {
       'handler' => 'views_handler_filter_valid_fia'
     ),
   );
+
+  $data['node']['fb_instant_articles_selected_views'] = array(
+    'title' => t('Selected Content for Facebook instant articles feed'),
+    'filter' => array(
+      'title' => t('Selected Content for Facebook instant articles filter'),
+      'help' => t('Filter for items that where selected for beeing shown in facebook instant articles feed.'),
```

Comment:
Nitpick, spelling: "Filter for items that were selected for being shown in Facebook Instant Articles feed."